astropy/utils/data.py

```diff
@@ -1226,69 +1303,71 @@ def _deltemps():
         while len(_tempfilestodel) > 0:
             fn = _tempfilestodel.pop()
             if os.path.isfile(fn):
-                os.remove(fn)
+                try:
+                    os.remove(fn)
+                except OSError:
+                    # oh well we tried
+                    # could be held open by some process, on Windows
```

Comment:
I don't have a Windows machine, but yes, the file goes where Windows thinks temporary files go. Whether Windows ever cleans that up I don't know. (This has been astropy behaviour since forever, except that it would crash on exit instead of exiting cleanly if this occurred.)